### PR TITLE
build: publish to npm

### PR DESCRIPTION
**Pull request by nowseemee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,15 +14,19 @@
     "bootstrap": "lerna bootstrap",
     "postbootstrap": "lerna run generate",
     "new-version": "lerna version --conventional-commits --no-push",
-    "force-version": "lerna version --conventional-commits --force-publish --no-push",
-    "format": "lerna run format"
+    "force-version": "lerna version --conventional-commits --force-publish",
+    "format": "lerna run format",
+    "publish-telekom": "lerna publish -y from-package --no-git-tag-version --no-push --force-publish @telekom/scale-components,@telekom/scale-components-react,@telekom/scale-components-angular,@telekom/scale-components-vue,@telekom/scale-design-tokens",
+    "prepare-neutral": "node scripts/prepare-neutral-release.js && git commit . -m 'chore: temp commit'",
```
Comment (acstll) [inline package.json:20]: what happens to this commit? 🤔 does it get lost/piled up in the CI machine?

**Review by acstll — COMMENTED** [review]
everything else looks 🆗 to me

File: package.json (hunk 1 above)
Comment (nowseemee) [reply package.json:20]: it goes to the void 🤷🏻

Comment (nowseemee) [reply package.json:20]: would you like to keep it? if yes, where would you store it?

Comment (acstll) [reply package.json:20]: nope, the opposite, if it goes into the void then it's fine 😄

**Pull request merged**